<launch>
  <param name="states_file" type="string" value="$(find opam)/data/states.txt"/>
  <param name="robot_description" textfile="$(find opam)/urdf/matrice.urdf"/>
  <param name="controls_file" type="string" value="$(find opam)/data/controls.txt"/>
  <param name="wFile" type="string" value="$(find opam)/config/W.txt"/>
  <param name="qFile" type="string" value="$(find opam)/config/Q2.txt"/>

  <rosparam command="load" file="$(find opam)/config/params.yaml"/>

  <node type="controller_test" pkg="opam" name="controller_test" output="screen"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find urdf_tutorial)/urdf.rviz" required="true" />
</launch>
